test/source/tests/page-recipe/settings-page-recipe.ts

```diff
@@ -168,4 +168,15 @@ export class SettingsPageRecipe extends PageRecipe {
     await settingsPage.waitForContent('@container-settings-keys-list', fp); // confirm key successfully loaded
     await settingsPage.close();
   };
+
+  public static checkContactKey = async (
+    contactsFrame: ControllableFrame,
+    keyType: 'OPENPGP' | 'X509',
+    keyStatus: 'ACTIVE' | 'SIGN ONLY' | 'EXPIRED' | 'UNUSABLE' | 'REVOKED',
+    fingerprint: string
+  ) => {
+    await contactsFrame.waitForContent('@page-contacts', fingerprint);
+    await contactsFrame.waitForContent('@container-contact-key-type', keyType);
+    await contactsFrame.waitForContent('@container-key-status', keyStatus);
```

Comment:
it seems we can get incorrect test results when there are several keys on contacts screen. as when checking `await contactsFrame.waitForContent('@container-contact-key-type', keyType);` we don't explicitly specify which key we're checking.

the same for `await contactsFrame.waitForContent('@container-key-status', keyStatus);`

to fix this we need to explicitly specify key id in `data-test` property (like it's currently done for `action-show-pubkey-${keyid}`

then we'll be able to check status and type for correct key, like this `await contactsFrame.waitForContent('@container-key-status-${keyId}', keyStatus);`